src/test/regress/bin/normalize.sed

```diff
@@ -176,6 +176,14 @@ s/(->.*Scan on\ +)(.*)(_[0-9]+)(_[0-9]+) \2(_[0-9]+|_xxx)?/\1\2\3\4 \2_xxx/g
 s/(partitioning_hash_join_test)(_[0-9]|_xxx)?(\.[a-zA-Z]+)/\1_xxx\3/g
 s/(partitioning_hash_test)(_[0-9]|_xxx)?(\.[a-zA-Z]+)/\1_xxx\3/g
 
+# multi_foreign_key_relation_graph.out
+# suppress shardId's in drop constraint logs as the order of cascaded objects might be flaky
+# e.g: drop cascades to constraint fkey_6_3000081 on table fkey_graph.distributed_table_1_3000081
+s/(drop cascades to constraint fkey_[0-9]+_)([0-9]+)( on table fkey_graph\..+_)\2/\1xxx\3xxx/g
+
+# remove DEBUG1 messages specific to pg11 that are about foreign key validation
```

Comment:
Is it? If so, why didn't we have pg 11/pg12 output files for the tests?

Reply:
true, seems a little suspicious, I will look into that again

Reply:
Looked into pg source code, seems like they stopped validating foreign key constraints for partitioned tables starting from pg12, by adding following check:

```c
		/* Relations without storage may be ignored here too */
		if (!RELKIND_HAS_STORAGE(tab->relkind))
			continue;
```

where,

```c
#define RELKIND_HAS_STORAGE(relkind) \
	((relkind) == RELKIND_RELATION || \
	 (relkind) == RELKIND_INDEX || \
	 (relkind) == RELKIND_SEQUENCE || \
	 (relkind) == RELKIND_TOASTVALUE || \
	 (relkind) == RELKIND_MATVIEW)
```

https://github.com/postgres/postgres/commit/46e3442c9ec858071d60a1c0fae2e9868aeaa0c8#diff-d373fb790605aded212c2e3a0fbe8c015d110077ff46c07d0a6d6647a2d7e928L4647-L4667

Reply:
Ok, so we are adding the normalize rule as we add a new test with partitioned tables, thanks for the clarification. (And thanks for @pykello  for fixing the  issue on PG :))